scripts/run-migrations.ts

```diff
@@ -215,6 +220,72 @@ class MigrationRunner {
     execSync(`rm -rf ${tempMigrationsDir}`, { cwd: process.cwd() });
   }
 
+  private async displayIntrospectionFiles(): Promise<void> {
```

Comment:
this method is the opposite of what we wanted. We want to print just the new migration file after the initial introspection file